libc/utils/docgen/setjmp.json

```diff
@@ -0,0 +1,15 @@
+{
+  "macros": {
+    "__STDC_VERSION_SETJMP_H__": {
+      "defined": "7.13.2"
+    },
+    "setjmp": {
+      "defined": "7.13.1.1"
```

Comment:
`setjmp` is a function, not a macro. Mind moving it down to that section?

Reply:
Interesting, it sounds like it is a macro??:

> 7.13.1.1 The `setjmp` macro

Reply:
Looks like it might need to be switched to a macro and not exposed in the spec and entrypoints as a function?

Reply:
oh jeez! I'd better read the spec! TIL